src/components/MoneyRequestHeader.js

```diff
@@ -187,14 +187,24 @@ const MoneyRequestHeader = (props) => {
                         titleIcon={Expensicons.Checkmark}
                         description={`${props.translate('iou.amount')} • ${props.translate('iou.cash')}${isSettled ? ` • ${props.translate('iou.settledExpensify')}` : ''}`}
                         titleStyle={styles.newKansasLarge}
+                        disabled={isSettled}
+                        // Note: These options are temporarily disabled while we figure out the required API changes
```

Comment:
Just so we don't forget, is there an issue for this?

Reply:
Created one here - https://github.com/Expensify/Expensify/issues/291689

Reply:
ty